README.md

```diff
@@ -1,246 +1,84 @@
+## Features
 
-```
-cd eleventy-decap-tailwind-template
-```
+- **Eleventy**: A static site generator that allows you to create dynamic and flexible static websites using various templating languages.
+- **Tailwind CSS**: A highly customizable CSS framework that simplifies designing responsive and visually appealing user interfaces.
+- **Decap CMS**: An intuitive content management system that enables non-technical users to manage website content seamlessly.
 
-## 2. Install Node.js and npm
+## Getting Started
 
-This template requires Node.js version 14 or higher in order to run. You can check whether you have Node installed by running `node --version`
-in the terminal. If the command is not found or the version number is less than 14, you will need to download and install the software. You can
-download the latest version of Node.js and npm with the following command:
+### Prerequisites
 
-```
-npm install -g npm
-```
+Before you begin, ensure you have the following installed:
 
+- Node.js (version 14 or later)
+- npm (Node Package Manager)
 
-## 3. Install Eleventy
+### Installation
 
-We can now install the static site generator Eleventy. `@11ty/eleventy` is published on npm, so we can install and save it
-to our project's `package.json` using:
+1. Clone this repository to your local machine:
 
-```
-npm install @11ty/eleventy --save-dev
-```
+   ```
+   git clone https://github.com/kindsys/dsc-11ty-cms.git
+   ```
+2. Navigate to the project directory:
 
+   ```
+   cd dsc-11ty-cms
+   ```
+3. Install the project dependencies:
 
-## 4. Run the template
+   ```
+   npm install
+   ```
+### Configuration
 
-Use the following script to test your installations:
+#### Eleventy Configuration:
+- Open the `eleventy.js` file and configure template engines, I/O directories, collections, filters, shortcodes, and more.
 
-```
-npm run build
-```
+#### Decap CMS Configuration:
+- Open the `admin/config.yml` file and configure settings such as the content directory, collections, and more.
 
-There should be multiple lines of output, but the last line should look like this if the run is successful:
+## Usage
 
-```
-[11ty] Wrote 13 files in 0.18 seconds (v2.0.1)
-```
+### Development
 
-## 5. View the page
+To start the development server and build your Eleventy project with Tailwind CSS, run:
 
-Now we can view our page on a local web server by using the `serve` script:
+ ```
+ npm run serve
+ ```
 
-```
-npm run serve
-```
+ It first runs the "clean" command to remove the _site directory. Then, it runs the commands that start local development servers using webpack and  Eleventy concurrently.
 
-Part of your output in the terminal should look like this:
 
-```
-[11ty] Server at http://localhost:8080/
-```
+### Content Management
 
-# Deployment
+To access the CMS open your web browser and navigate to `http://localhost:8080/admin` to access the Decap CMS interface. Here, you can manage your website's content.
```

Comment:
Would suggest elaborating here on how to set up the local proxy to use the CMS locally, e.g.
1. uncommenting the `local_backend: true ` and git-gateway backend in the config.yml file 
2. running `npx netlify-cms-proxy-server` alongside the `npm run serve` command

which should allow them to go to localhost:8080/admin without authenticating and access the CMS